src/motion/types.ts

```diff
@@ -255,29 +255,44 @@ export interface AnimationProps {
     transition?: Transition
 
     /**
-     * If `positionTransition` is defined, the component will automatically animate any changes to its layout
-     * relative to its nearest positioned parent.
+     * @layout
```

Comment:
What does `@layout` do? Shouldn't this be `@library`?

Reply:
Yes - doh